bot.go

```diff
@@ -1062,7 +1061,7 @@ func (b *Bot) StopLiveLocation(message Editable, options ...interface{}) (*Messa
 	messageID, chatID := message.MessageSig()
 
 	params := map[string]string{
-		"chat_id":    fmt.Sprintf("%d", chatID),
```

Comment:
Why `FormatInt` and not `Itoa`?

Reply:
@stek29 
Because `Message.MessageSig()` returns `int64`, but `strconv.Itoa` accepts only `int`. Anyway, `strconv.Itoa` [uses](https://golang.org/src/strconv/itoa.go?s=1028:1051#L24) `strconv.FormatInt`:
```go
// Itoa is equivalent to FormatInt(int64(i), 10).
func Itoa(i int) string {
	return FormatInt(int64(i), 10)
}
```